crates/bevy_render/src/render_resource/uniform_vec.rs

```diff
@@ -119,48 +144,63 @@ impl<T: AsStd140> Default for DynamicUniformVec<T> {
 }
 
 impl<T: AsStd140> DynamicUniformVec<T> {
+    /// Gets the reference to the underlying buffer, if one has been allocated.
     #[inline]
-    pub fn uniform_buffer(&self) -> Option<&Buffer> {
-        self.uniform_vec.uniform_buffer()
+    pub fn buffer(&self) -> Option<&Buffer> {
+        self.uniform_vec.buffer()
     }
 
+    /// Creates a binding for the underlying buffer.
+    /// Returns `None` if no buffer has been allocated.
     #[inline]
     pub fn binding(&self) -> Option<BindingResource> {
         self.uniform_vec.binding()
     }
 
+    /// Gets the capacity of the underlying buffer, in bytes.
```

Comment:
`buffer_capacity()` will likely be removed but just noting that it doesn’t return bytes.